en/admin/control.rst

```diff
@@ -2308,6 +2284,56 @@ Additionally, The server's PID, port number, and the applied JVM option are show
     -Xrs
     -------------------------------------------------
 
+
+.. _cubrid-javasp-with-server:
+
+Starting the CUBRID Java SP Server together when the database server starts
+------------------------------------------------------------------------------------------
+
+| If **java_stored_procedure** set to yes for the corresponding database, 
+| When the database server starts/stops, the Java stored procedure server is started/stopped.
+| The following is an example of Java stored procedure server is started together when the database server starts.
+
+::
+
+    # cubrid.conf
+
+    ...
+
+    [@demodb]
+    java_stored_procedure=yes
+    
+    [@testdb]
+    java_stored_procedure=no
+
+    ...
+
+::
+
+    -- demodb's java_stored_procedure is set to yes
+    % cubrid server start demodb
+    
+    @ cubrid server start: demodb
+
+    This may take a long time depending on the amount of restore works to do.
```

Comment:
```suggestion
    This may take a long time depending on the amount of restored work.
```
'restore' is a verb, need to change it. to 'restored'.

Reply:
This message is just displayed by cubrid utility. We don't need to fix it.